packages/api-plugin-accounts/src/queries/filterSearchCustomers.js

```diff
@@ -0,0 +1,25 @@
+import generateFilterQuery from "@reactioncommerce/api-utils/generateFilterQuery.js";
+
+/**
+ * @name filterSearchCustomers
+ * @method
+ * @memberof GraphQL/Customers
+ * @summary Query the Accounts collection for a list of customers/accounts
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} conditions - object containing the filter conditions
+ * @param {String} shopId - shopID to filter by
+ * @returns {Promise<Object>} Accounts object Promise
+ */
+export default async function filterSearchCustomers(context, conditions, shopId) {
+  const { collections: { Accounts } } = context;
+
+  if (!shopId) {
+    throw new Error("shopId is required");
+  }
+  await context.validatePermissions("reaction:legacy:accounts", "read", { shopId });
+
+  const { filterQuery } = generateFilterQuery(context, "Account", conditions, shopId);
+
+  filterQuery.groups = { $in: [null, []] }; // filter out non-customer accounts
```

Comment:
Are we sure that this is a completely correct way to retrieve only customers?

Reply:
The existing query endpoint uses the same approach. Hence I re-used it
`packages/api-plugin-accounts/src/queries/customers.js`